operator/config/olm-catalog/ibm-spectrum-scale-csi-operator/2.6.0/ibm-spectrum-scale-csi-operator.v2.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,558 @@
+                      - key: kubernetes.io/arch
+                        operator: In
+                        values:
+                        - amd64
+                        - ppc64le
+                        - s390x
+              containers:
+              - args:
+                - --leaderElection=true
+                env:
+                - name: METRICS_BIND_ADDRESS
+                  value: "8383"
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: CSI_DRIVER_IMAGE
+                  value: cp.icr.io/cp/spectrum/scale/csi/ibm-spectrum-scale-csi-driver@sha256:695b7107c16fce429f559fc6e26686712f8670bfd1d051b85279efd48a7eb413
+                image: cp.icr.io/cp/spectrum/scale/csi/ibm-spectrum-scale-csi-operator@sha256:66c8c36e5e1f7f9095a0f6133c57b3598997f1d412cbe9c3ebbea75af2ccc534
+                imagePullPolicy: IfNotPresent
+                livenessProbe:
+                  exec:
+                    command:
+                    - ./health_check.sh
+                  initialDelaySeconds: 10
+                  periodSeconds: 30
+                name: operator
+                readinessProbe:
+                  exec:
+                    command:
+                    - ./health_check.sh
+                  initialDelaySeconds: 3
+                  periodSeconds: 1
+                resources:
+                  limits:
+                    cpu: 600m
+                    ephemeral-storage: 5Gi
+                    memory: 600Mi
+                  requests:
+                    cpu: 50m
+                    ephemeral-storage: 5Gi
+                    memory: 50Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                    - ALL
+                  privileged: false
+                  readOnlyRootFilesystem: false
+                  runAsNonRoot: true
+                  runAsUser: 1001
+              serviceAccountName: ibm-spectrum-scale-csi-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - IBM
+  - GPFS
+  - Spectrum Scale
+  - CSI Driver
+  labels:
+    operator: ibm-spectrum-scale-csi-operator
+  links:
+  - name: IBM CSI Spectrum Scale Documentation
+    url: https://www.ibm.com/docs/en/spectrum-scale-csi
+  - name: CSI Developer Documentation
+    url: https://kubernetes-csi.github.io/docs/
+  maintainers:
+  - email: madhu.punjabi@in.ibm.com
+    name: Madhu Thorat
+  - email: amdabhad@in.ibm.com
+    name: Amol Dabhade
+  - email: Nitish.Kumar4@ibm.com
+    name: Nitish Kumar
+  maturity: alpha
+  provider:
+    name: IBM
+  replaces: ibm-spectrum-scale-csi-operator.v2.5.0
```

Comment:
should this be 2.5.1? updating `spec.replaces` in /operator/config/manifests/bases/ibm-spectrum..yaml file and executing `make bundle` should update this where ever required.